### PR TITLE
added on hover effects to profile icon 

### LINKED ISSUE #5194: Add on hover effect to profile icon on navbar
# Description
In #5093 the navbar UI was updated to add a fontawesome icon however this shows no animation on hover

# Expected
The icon shows animation on hover like this.

iconcss

### PR DESCRIPTION
**Pull request by ananyaarun**
Fixes #5194

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

iconcss

Thanks!

### CODE REVIEW

**Comment by ananyaarun** [comment]
Wrt issue #5194 . I have added the hover effect on profile icon. 
@publiclab/plots2-reviewers Kindly review.

File: app/views/layouts/_header.html.erb [hunk 1]
```diff
@@ -1,3 +1,5 @@
+<link rel="stylesheet" href="font-awesome-animation.min.css">
```
Comment (gauravano) [inline app/views/layouts/_header.html.erb]: I don't think this is needed?

File: app/views/layouts/_header.html.erb [hunk 2]
```diff
@@ -174,3 +178,17 @@
...
+ 
+  .profile-icon
+  {
+       cursor:pointer;
+    }
+ 
+    .profile-icon:hover .fa-user-circle:before
+    {
+      opacity:0.5;
+    }
```
Comment (gauravano) [inline app/views/layouts/_header.html.erb]: Please do proper indentation here

File: app/views/layouts/_header.html.erb (hunk 1 above)
Comment (ananyaarun) [reply app/views/layouts/_header.html.erb]: Oops my bad ! I initially integrated it with the fontawesome CSS but this seemed to work better. Will remove that line right away

File: app/views/layouts/_header.html.erb (hunk 2 above)
Comment (ananyaarun) [reply app/views/layouts/_header.html.erb]: Sure i'll take care of that

**Comment by ananyaarun** [comment]
@gauravano , I made the required changes.

**Comment by jywarren** [comment]
Thank you!!!

**Pull request merged**